lib/eex/lib/eex/compiler.ex

```diff
@@ -13,16 +13,19 @@ defmodule EEx.Compiler do
   def compile(source, opts) when is_binary(source) and is_list(opts) do
     file = opts[:file] || "nofile"
     line = opts[:line] || 1
+    column = 1
     trim = opts[:trim] || false
 
-    case EEx.Tokenizer.tokenize(source, line, trim: trim) do
+    case EEx.Tokenizer.tokenize(source, line, column, trim: trim) do
       {:ok, tokens} ->
         state = %{
           engine: opts[:engine] || @default_engine,
           file: file,
           line: line,
           quoted: [],
-          start_line: nil
+          start_line: nil,
+          start_column: nil,
+          columns: Code.get_compiler_option(:parser_options)[:columns] == true
```

Comment:
Maybe we should keep all `parser_options` here and pass all of them down to the `string_to_quoted!` call.